internal/cli/certificate.go

```diff
@@ -49,6 +49,17 @@ type CertificateOptions struct {
 	SignerName string
 }
 
+type enrollmentService struct {
+	baseclient.Config
+
+	// EnrollmentUIEndpoint is the address of the device enrollment UI
+	EnrollmentUIEndpoint string `json:"enrollment-ui-endpoint,omitempty"`
```

Comment:
This will be a problem eventually if the config format is changed @lkatalin *

What gregory is trying to fix here is that otherwise the CLI imported the agent, pulling in a lot of agent code that then needed windows compatibility.

Could be beneficial to move those structures into a common place outside of the agent tree?

Reply:
I think I see the goal and potential issue, and agree with @mangelajo it would be best if the agent config is pulled in both here and to the `internal/agent` from a shared location that is separated from the other `agent` code into its own package.

Reply:
Done